src/Stratis.Bitcoin.Features.Miner/Controllers/MiningController.cs

```diff
@@ -70,8 +59,8 @@ public IActionResult Generate([FromBody]MiningRequest request)
 
             try
             {
-                if (this.network.Consensus.IsProofOfStake)
-                    return ErrorHelpers.BuildErrorResponse(HttpStatusCode.MethodNotAllowed, "Method not allowed", "Method not available for Proof of Work");
+                if (this.network.Consensus.IsProofOfStake && (this.consensusManager.Tip.Height > this.network.Consensus.LastPOWBlock))
+                    return ErrorHelpers.BuildErrorResponse(HttpStatusCode.MethodNotAllowed, "Method not allowed", $"This is a POS node and it's consensus tip is higher than the allowed LastPowBlock height of {this.network.Consensus.LastPOWBlock}");
```

Comment:
it's => its
This message contains too many details IMO.
"Mining is not allowed past block {this.network.Consensus.LastPOWBlock}." is simpler.

Reply:
I want to include the fact this will only happen if the node is POS, so have amended to:
`"This is a POS node and mining is not allowed past block {0}"`